src/main/kotlin/org/ptss/support/infrastructure/persistence/entities/CategoryEntity.kt

```diff
@@ -0,0 +1,63 @@
+package org.ptss.support.infrastructure.persistence.entities
+
+import jakarta.persistence.*
+import org.ptss.support.domain.models.Category
+import java.time.Instant
+import java.util.*
+
+@Entity
+@Table(name = "categories")
+class CategoryEntity {
+    @Id
+    @Column(nullable = false, unique = true)
+    lateinit var category: String // Primary key
+
+    @Column(name = "group_id", nullable = false, columnDefinition = "UUID")
```

Comment:
you're using hibernate, this already handles the uuid definition in the database, you can thus omit columnDefinition = "UUID"
```suggestion
    @Column(name = "group_id", nullable = false)
```